### PR TITLE
ahost: add page

### PR DESCRIPTION
**Pull request by shubham-kshetre**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by suyashbhawsar — APPROVED** [review]
Looks good to me!

**Review by suyashbhawsar — APPROVED** [review]

File: pages/linux/ahost.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# ahost
+
+> DNS lookup utility to display the A or AAAA record linked with a hostname or IP address.
+> More information: <https://manned.org/ahost>.
+
+- Get A or AAAA record associated with a hostname or IP address:
```
Comment (CleanMachine1) [inline pages/linux/ahost.md:6]:
```suggestion
- Get an `A`or `AAAA` record associated with a hostname or IP address:
```

File: pages/linux/ahost.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`ahost {{example.com}}`
+
+- Display some extra debugging output:
+
+`ahost -d {{example.com}}`
+
+- Display the record with specified type:
+
+`ahost -t {{type}} {{example.com}}`
```
Comment (CleanMachine1) [inline pages/linux/ahost.md:16]:
```suggestion
`ahost -t {{a|aaaa}} {{example.com}}`
```

**Review by CleanMachine1 — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution

File: pages/linux/ahost.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+`ahost {{example.com}}`
+
+- Display some extra debugging output:
+
+`ahost -d {{example.com}}`
+
+- Display the record with specified type:
+
+`ahost -t {{a|aaaa}} {{example.com}}`
```
Comment (kbdharun) [inline pages/linux/ahost.md:16]:
```suggestion
`ahost -t {{a|aaaa|u}} {{example.com}}`
```
Just a small suggestion to this page. 
>  If the type is "a", print the A record (default).  If the type is "aaaa",
              print the AAAA record.  If the type is "u", look for either AAAA or A
              record (in that order).
Ref: <https://manned.org/ahost>.

File: pages/linux/ahost.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+- Print an `A` or `AAAA` record associated with a hostname or IP address:
+
+`ahost {{example.com}}`
+
+- Display some extra debugging output:
+
+`ahost -d {{example.com}}`
+
+- Display the record with specified type:
```
Comment (kbdharun) [inline pages/linux/ahost.md:14]:
```suggestion
- Display the record with a specified type:
```
My last suggestion before merge.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**